lib/bolt/inventory/inventory2.rb

```diff
@@ -147,7 +121,6 @@ def config_plugin(data)
       private :config_plugin
 
       # Pass a target to get_targets for a public version of this
-      # Should this reconfigure configured targets?
       def update_target(target)
         data = @groups.data_for(target.name)
         data ||= {}
```

Comment:
I'm not sure when this data would be nil, since targets should always at least be part of the 'all' group.

Reply:
This can be nil. 
As a trivial example, with an empty v2 inventory. Run `bolt command run whoami -t localhost`, the group `data_for` method will return nil.